lib/buffer.js

```diff
@@ -1207,10 +1207,10 @@ if (internalBinding('config').hasIntl) {
       return result;
 
     const code = icuErrName(result);
-    // eslint-disable-next-line no-restricted-syntax
-    const err = new Error(`Unable to transcode Buffer [${code}]`);
-    err.code = code;
-    err.errno = result;
+    const err = genericNodeError(
+      `Unable to transcode Buffer [${code}]`,
+      { code: code, errno: result }
+    );
     throw err;
```

Comment:
Not saying it's a good idea, but allowed as per https://github.com/nodejs/node/pull/41586 as far as I can tell (98 columns):

```suggestion
    throw genericNodeError(`Unable to transcode Buffer [${code}]`, { code: code, errno: result });
```